src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -3,6 +3,8 @@ import {makeStyles} from '@material-ui/core/styles';
 import Typography from "@material-ui/core/Typography";
 import axios from 'axios';
 
+const authToken = localStorage.getItem('authorization_token');
```

Comment:
With this approach If I load your app i get the token, but while testing I can change localstorage in dev tools, so I have to reload the page. It good that I found this thing but some other testers can just think that your app doesn't work properly.

To get rid of this you can just place this line to `uploadFile` method.

Reply:
Thank you. Fixed that. Also refactored header for 401 error then no token and 403 for wrong.